plugins/needs-review/webMonitor/trunk/webMonitorServer.pm

```diff
@@ -380,7 +380,7 @@ sub request {
 	my ($i, $name, $class, $lvl, $title, $online, $ID, $charID);
 	my (@listMemberIndex, @listMemberName, @listMemberClass, @listMemberLvl, @listMemberTitle, @listMemberOnline, @listMemberID, @listMemberCharID);
 	
-	if (defined @{$guild{member}}) {
+	if (defined $guild{member}) {

```

Comment:
`defined @{$guild{member}}` is equivalent to `$guild{member} && @{$guild{member}}`